server/src/Types.hs

```diff
@@ -0,0 +1,86 @@
+module Types (
+  AppM (AppM),
+  Env (..),
+  Cbor (..),
+  Fee (..),
+  FeeEstimateError (..),
+  CardanoBrowserServerError (..),
+  newEnvIO,
+) where
+
+import Cardano.Api.Shelley qualified as Shelley
+import Cardano.Binary qualified as Cbor
+import Control.Exception (Exception)
+import Control.Monad.Catch (MonadThrow)
+import Control.Monad.IO.Class (MonadIO)
+import Control.Monad.Reader (MonadReader, ReaderT)
+import Data.Aeson (FromJSON, ToJSON (..))
+import Data.Aeson qualified as Aeson
+import Data.Aeson.Encoding qualified as Aeson
```

Comment:
Entirely optional but there's a "duplicate" import here too.

Reply:
Will change this too